app/components/tab_navigation/view.html.erb

```diff
@@ -0,0 +1,13 @@
+<nav class="app-tab-navigation" aria-label="Sub navigation">
+  <ul class="app-tab-navigation__list">
+    <% items.each do |item| %>
+      <li class="app-tab-navigation__item">
+        <% if item.fetch(:current, false) || current_page?(item.fetch(:url)) %>
+          <%= govuk_link_to item[:name], item[:url], class: "app-tab-navigation__link", aria: { current: "page" } %>
+        <% else %>
+          <%= govuk_link_to item[:name], item[:url], class: "app-tab-navigation__link" %>
+        <% end %>
+      </li>
+    <% end %>
+  </ul>
+</nav>
```

Comment:
end of file

Reply:
done https://github.com/DFE-Digital/register-trainee-teachers/pull/215/commits/c62c62a4a5a29ab52479e7579d34265dee9f7cd0